clib.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nuklear",
-  "version": "4.10.6",
+  "version": "4.10.7",
```

Comment:
Thanks! Since this is part of the demos, likely don't need to bump the version number :+1: 
``` suggestion
  "version": "4.10.6",
```